actions/list.action.ts

```diff
@@ -1,39 +1,52 @@
 import * as chalk from 'chalk';
 import * as fs from 'fs';
-import { platform, release } from 'os';
-import osName = require('os-name');
 import { join } from 'path';
-import {
-  AbstractPackageManager,
-  PackageManagerFactory,
-} from '../lib/package-managers';
-import { BANNER, MESSAGES } from '../lib/ui';
+import { AbstractPackageManager, PackageManagerFactory } from '../lib/package-managers';
 import { AbstractAction } from './abstract.action';
-
+import { MESSAGES } from '../lib/ui';
 
 export class ListAction extends AbstractAction {
   private manager!: AbstractPackageManager;
+  private specFilePath: string;
+
+  constructor() {
+    super();
+    this.specFilePath = 'spec.yaml'; // Default spec file path if not found in nest-cli.json
```

Comment:
Why is this path hardcoded?

Reply:
default spec.yaml path if no path is found inside nest-cli.json

Reply:
also the spec file will be generated automatically by the compiler when the project is being bootstrap.